source/client-side-operations-timeout/client-side-operations-timeout.md

```diff
@@ -427,6 +427,16 @@ check the command document for the presence of a `maxTimeMS` field.
 
 See [runCommand behavior](#runcommand-behavior).
 
+### Explain Helpers
+
+If a driver provides an explain helper, drivers MUST take care to ensure that timeoutMS is correctly applied to the
+top-level explain command, when specified. Care should be taken by drivers with a fluent API - the following example
```

Comment:
"top-level explain command" - Is that referencing how the "find" is nested under an `explain` field? there is no non top-level explain right? 

Maybe remove top-level? something along the lines of "when timeoutMS is used with explain API helpers, timeoutMS must apply to the explain command." Because IIUC the concern here is that it may be ignored accidentally if the implementation is set up in such a way to only apply it to the command _being_ explained right?